README.md

```diff
@@ -212,6 +222,17 @@ Build failed in WPF projects [microsoft/CsWin32#7](https://github.com/microsoft/
   [#dotnetcore]: https://twitter.com/hashtag/dotnetcore?src=hashtag_click
   [#analyzers]: https://twitter.com/hashtag/analyzers?src=hashtag_click
 
+Reference local projects or embed NuGet packages to source generator assemblies [dotnet/roslyn#47517](https://github.com/dotnet/roslyn/discussions/47517#discussioncomment-64145):
```

Comment:
```suggestion
Reference local projects or embed NuGet packages to source generator assemblies [dotnet/roslyn#47517](https://github.com/dotnet/roslyn/discussions/47517#discussioncomment-64145):

⚠ Please be aware that this may result in crashes, when another generator or SDK component loads such an assembly with lower version. If you can, please avoid embedding additional DLLs/packages.
```